components/apimgt/org.wso2.carbon.apimgt.rest.api.util/src/main/java/org/wso2/carbon/apimgt/rest/api/util/utils/JWTAuthenticationUtils.java

```diff
@@ -19,6 +19,8 @@
 import org.apache.cxf.message.*;
 import java.util.HashMap;
 
+import static org.wso2.carbon.apimgt.rest.api.common.RestApiConstants.USER_REST_API_SCOPES;
```

Comment:
Any necessity to use import static? Shall we please only import RestApiConstants normally and refer as RestApiConstants.USER_REST_API_SCOPES in the required place?

Reply:
Sometimes static imports are causing unwanted problems in OSGi space, that is why I wanted to avoid this.

Reply:
Those constants are defined as static constants.

Reply:
Fixed via : https://github.com/wso2/carbon-apimgt/pull/12038/commits/d84e45e68abc9e3f468efea943bfa96fd1c5ff27